src/modules/form/components/client/addresses/SingleAddressInput.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from 'react';
+
+import { DynamicInputCommonProps } from '../../../types';
+import { AddressInputType } from '../types';
+import { useRenderLastUpdated } from '../useRenderLastUpdated';
+import { createInitialValue } from '../util';
+
+import AddressInput from './AddressInput';
+
+import SingleInputContainer from '@/modules/form/components/client/SingleInputContainer';
+import { ClientAddressFieldsFragmentDoc } from '@/types/gqlTypes';
+
+interface Props extends DynamicInputCommonProps {
+  value?: AddressInputType[];
+  onChange: (value: AddressInputType[]) => void;
+}
+
+const initialValue: AddressInputType[] = [];
+const SingleAddressInput: React.FC<Props> = ({
+  id,
+  value,
+  onChange,
+  label,
+}) => {
+  const values = value ? value : initialValue;
+
+  useEffect(() => {
+    if (values.length == 0) {
+      onChange([createInitialValue()]);
+    }
+  }, [values, onChange]);
+
+  const { renderMetadata } = useRenderLastUpdated(
+    'ClientAddress',
+    ClientAddressFieldsFragmentDoc,
+    'ClientAddressFields'
+  );
+
+  return (
+    <SingleInputContainer
+      id={id}
+      values={values}
+      valueKey={(addrValue) => addrValue._key || addrValue.id || ''}
+      renderChild={(addrValue, idx) => {
+        return (
+          <AddressInput
+            value={addrValue}
+            onChange={(val) => {
+              const copied = [...values];
+              copied[idx] = val;
+              onChange(copied);
+            }}
```

Comment:
Can we pass a `hideType` `hideNote` here (or from DynamicField), or something similar?

I think when collecting an individual address, it's collected in a context that implies the type.. at least that is true for Move In. 

Ideally:

<img width="632" alt="Screenshot 2023-11-01 at 9 53 33 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/016ccb49-3a39-448a-b86b-a849c47ab1a5">

Reply:
> Can we pass a hideType hideNote here (or from DynamicField), or something similar?

Sure! I added a variant to address input. I also renamed "SingleAddressInput" to "SimpleAddressInput" to be more descriptive/accurate since it does support multiple values.